webpack.config.js

```diff
@@ -18,37 +18,20 @@ const { getIfUtils, removeEmpty } = require( 'webpack-config-utils' );
 
 
 module.exports = ( env ) => {
-  const { ifProd, ifNotProd, ifTest, ifNotTest } = getIfUtils( env );
+  const { ifProd, ifNotProd, ifTest, ifDev, ifSite } = getIfUtils( env, ['prod', 'test', 'dev', 'site'] );
+
+  const packagePath = resolve( './packages', env.element || '' );
```

Comment:
this doesn't work. Pls don't always copy paste what I write in review :D Im not unit tested because I'm human :D 

there is context needed, again we can make things DRY

```js
const context = resolve(__dirname);
const packagePath = resolve( context, './packages', env.element || '' );

return {
   context,
   entry: getEntryPointConfig( packagePath, { ... } ),
  // ...
   
```